src/ZLogger.MessagePack/ZLogger.MessagePack.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This csproj file is newly added.
- I cut out the common parts with ZLogger.csproj into [Directory.build.props](https://github.com/Cysharp/ZLogger/pull/76/files#diff-5c1900dca99df01c3ddc3a884282164b04deee76e79b176e2d34cfa47f3b8907)
- 📝 This is something I would like to release as a separate package, do I need to configure anything else? (I am not too sure).